integreat_cms/cms/templates/pages/page_pdf.html

```diff
@@ -102,11 +102,11 @@ <h1 id="title_page">{{ language.table_of_contents }}</h1>
                 <h1 class="level-{{ page.depth|add:"-1" }}">{{ page_translation.title }}</h1>
                 <div class="content">
                     {% if page.mirrored_page_first %}
-                        {{ page_translation.mirrored_translation_text|pdf_strip_fontstyles|safe }}
+                        {{ page_translation.mirrored_translation_text|pdf_strip_fontstyles|pdf_truncate_links:50|safe }}
```

Comment:
Questionable thing, but we could probably allow more symbols.
Maybe 100?
So that links that do not exceed line length are left as is.

Reply:
Yeah, I don't really have a clear opinion on that one, as I can see pros and cons in both cases. 
On the one hand more links might be truncated, when only 50 chars are allowed, on the other hand shorter links read nicer (we might also prevent weird looking line breaks etc) and with longer links the chance of edge cases might become bigger (e.g. what if links are indented, e.g because of a bullet point list?)

Reply:
Bullet point is a good point 😀
But I found one more case to think of:
If the link has a normal text, and this text is longer than 50 characters, it is truncated too
![image](https://user-images.githubusercontent.com/115008338/228364888-78cc0fb2-ad5e-45b7-842f-2f00babeb825.png)

Reply:
How about sticking with 50 chars but only truncate the specific words of the text?
![Screenshot 2023-03-28 at 23-52-01 Integreat - Englisch - Daily Life pdf](https://user-images.githubusercontent.com/16021269/228375103-192c46a5-3008-4263-a7ff-8fa628ed0d85.png)

Reply:
Yes, I like this solution.